### DISCUSSION TITLE
npm version is 0.14.4 but latest git version is 1.14.0

### DISCUSSION THREAD

**Discussion by BarShahino** [nextstrain/nextclade#771] [Help]
Hi, will appreciate your kind help with the following issue: 
I'm installing nextclade via  npm install --global @neherlab/nextclade but getting version 0.14.4 (which also looks like the latest here https://www.npmjs.com/package/@neherlab/nextclade/v/0.14.4)
However, on the nextclade git, it looks like the latest version is 1.14.0
is the npm repo deprecated? is there a way that I can get an update of it?
many thanks!

**Comment by ivan-aksamentov** [comment]
Hi @BarShahino,

Indeed the npm version of Nextclade is deprecated. And a long time ago (in terms of pandemic research anyways). There will be no updates to versions 0.x.

The data it uses is very old, it may produce results that are not correct for the latest SARS-CoV-2 strains. We don't recommend you to use it.

Also you are missing out on a lot of new features.

The usage of Nextclade 1.x is described in the documentation: https://docs.nextstrain.org/projects/nextclade/en/stable/user/nextclade-cli.html

Let us know if something is unclear there, or if you have more questions.

**Reply by BarShahino** [reply]
Hi @ivan-aksamentov ,
Thank you for your quick answer.
I'll try to use `conda install -c bioconda nextclade` instead.

Have a great one!
